### ISSUE TITLE
docker client cant connect to tcp://192.168.42.43:4243

### ISSUE THREAD

**Issue by lalyos** [fnichol/dvm#24]
Hi, 

dvm starts up nicely, i can start containers etc. after a while I cant reach it from docker client:

```
2014/03/12 16:04:52 dial tcp 192.168.42.43:4243: operation timed out
```

I can still connect to docker by: `dvm ssh`, and containers are still running.
## environment
- osx: 10.9
- dvm: 0.4.1
- docker (client side): 0.8.1, build a1598d1
- DOCKER_HOST=tcp://192.168.42.43:4243
## diagnostic

by checking the open ports, i can see virtualbox listening

```
lsof -i|grep 4243
VBoxHeadl 11971 lalyos   20u  IPv4 0x459e391648321dc7      0t0  TCP *:4243 (LISTEN)
```

**Comment by cwjohnston** [comment]
It seems like the IP address assignment for eth1 changes after about ~20 minutes of uptime. I've seen this happen with both virtualbox and vmware_fusion providers.

**Comment by fnichol** [comment]
We've hopefully tracked this down and fixed in #35. Yay!
